na-note.html

```diff
@@ -98,14 +138,6 @@
         }
       },
 
-      listeners: {
-        input: 'onInput'
-      },
-
-      observers: [
-        'updateTextContent(title, body)'
-      ],
-
       emptyClass: function(content, editable) {
```

Comment:
Looks like you could eliminate the `if` and simplify the method down to just:

`return !content ? 'hidden' : '';`